### PR TITLE
Feature/dof viz

### PR DESCRIPTION
**Pull request by ravi-rahul**
Added Dof Viz module
please review

### CODE REVIEW

File: include/tiovx_dof_viz_module.h [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+/** \brief DOF Viz module create helper function
+ *
+ * This DOF Viz create helper function will create the node using all the data objects created during the \ref app_init_dof_viz call.
+ * Internally calls \ref app_create_graph_dof_viz_write_output if en_out_dof_viz_write is set
+ *
+ * \param [in]     graph          OpenVX graph that has been created using \ref vxCreateGraph and where the DOF Viz node is created
+ * \param [in,out] obj            DOF Viz Module object which contains DOF Viz node and write node which are created in this function
+ * \param [in]     input_arr      flow vector input object array to DOF Viz node.
+ *
```
Comment (shyam-j) [inline include/tiovx_dof_viz_module.h:127]: Add comment about target_string

File: test/app_tiovx_dof_viz_module_test.c [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+{
+    vx_status status = VX_SUCCESS;
+
+    /* Create OpenVx Context */
+    obj->context = vxCreateContext();
+    status = vxGetStatus((vx_reference) obj->context);
+
+    if(status == VX_SUCCESS)
+    {
+        tivxHwaLoadKernels(obj->context);
```
Comment (shyam-j) [inline test/app_tiovx_dof_viz_module_test.c:135]: This is not required for visualization

File: test/app_tiovx_dof_viz_module_test.c [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+    }
+
+    return status;
+}
+
+static void app_deinit(AppObj *obj)
+{
+    tiovx_dof_viz_module_deinit(&obj->dofVizObj);
+
+    tivxHwaUnLoadKernels(obj->context);
```
Comment (shyam-j) [inline test/app_tiovx_dof_viz_module_test.c:169]: This is not required for visualization

File: include/tiovx_dof_viz_module.h (hunk 1 above)
Comment (ravi-rahul) [reply include/tiovx_dof_viz_module.h:128]: Done

**Comment by ravi-rahul** [comment]
Hi @shyam-j 

Can you please review and merge this

**Comment by shyam-j** [comment]
Please send PR to develop branch

**Pull request closed without merging**